### ISSUE TITLE
VuesticSimpleSelect: "displayValue.toLowerCase is not a function" inside v-for loop

### ISSUE THREAD

**Issue by amulchinock** [epicmaxco/vuestic-admin#413]
* **I'm submitting a ...**
  - [x] bug report

* **What is the current behavior?**
Attempting to use VuesticSimpleSelect component inside v-for causes console errors:

> [Vue warn]: Error in render: "TypeError: displayValue.toLowerCase is not a function"
> 
> found in
> 
> ---> <VuesticSimpleSelect> at src/vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect.vue
>        <VuesticWidget> at src/vuestic-theme/vuestic-components/vuestic-widget/VuesticWidget.vue
>          <HomeCarousel> at src/components/pages/cms/home-carousel.vue
>            <Anonymous>
>              <VuesticLayout> at src/vuestic-theme/vuestic-components/vuestic-layout/VuesticLayout.vue
>                <AppLayout> at src/components/admin/AppLayout.vue
>                  <App> at src/App.vue
>                    <Root>

* **Steps to reproduce**
See simplified code example:

```
<div v-for="(photo, index) in propertyPhotos" :key="index"">
<vuestic-simple-select
                label="Position"
                v-model="photo.selectedImagePosition"
                option-key="description"
                v-bind:options="imagePositions"
              />
</div>

<script>
import VueLiteBox from 'vue-litebox'

export default {
   name: 'my-component',
   
   data () {
      return {
         imagePositions: [
        {
          id: '1',
          description: '1'
        },
        {
          id: '2',
          description: '2'
        }
      ],

         propertyPhotos: [
        {
          src: 'URL REMOVED', // private, stripped out for this issue
          title: 'This image has a caption',
          selectedImagePosition: {
            id: '1',
            description: '1'
          }
        },
        {
          src: 'URL REMOVED', // private, stripped out for this issue
          title: 'This image also has a caption',
          selectedImagePosition: {
            id: '2',
            description: '2'
          }
        }
      ]
      }
   }
}
</script>
```

* **What is the expected behaviour?**
No console errors...

**Comment by smartapant** [comment]
@asvae @vlad-shusterman must be fixed

**Comment by asvae** [comment]
@amulchinock thanks. Will be fixed in 1.9.0.

For the time being replace `VuesticSimpleSelect.vue` with [new version](https://github.com/epicmaxco/vuestic-admin/blob/23e8b7c03ad2c27d3e5c95c57b783d6def69bd2b/src/vuestic-theme/vuestic-components/vuestic-simple-select/VuesticSimpleSelect.vue) to update.
